project.yml

```diff
@@ -0,0 +1,18 @@
+name: FontAwesome
+options:
+  bundleIdPrefix: com.alexusbergo
+  deploymentTarget:
+    iOS: "8.0"
+targets:
+  FontAwesome:
+    platform: iOS
+    scheme: {} # Generate default scheme
+    settings:
+      GCC_GENERATE_DEBUGGING_SYMBOLS: NO # Avoid downstream warning: "Redistributable static libraries should never be built with module debugging enabled"
+      MACH_O_TYPE: staticlib
+      MODULEMAP_FILE: FontAwesome/FontAwesome.modulemap
+    sources:
+      - path: FontAwesome
```

Comment:
shouldn't this handle the font file somehow? 🤔

Reply:
so long story short, our app already has a copy of `FontAwesome.ttf`. I tried removing that copy and installing this as a dynamic embedded framework but the font wasn't found when I manually tested the app. I suspect there's some stipulation like fonts need to be copied by the main app. Therefore I decided to keep the main app copy and just make this a static framework!

Reply:
🧐